packages/marketplace/src/sagas/__tests__/developer-detail.ts

```diff
@@ -0,0 +1,81 @@
+import developerDetailSaga, { developerDetailFetchDataFetch } from '../developer-detail'
+import ActionTypes from '@/constants/action-types'
+import { call, put, takeLatest } from '@redux-saga/core/effects'
+import {
+  developerDetailLoading,
+  developerDetailReceiveData,
+  developerDetailRequestDataFailure,
+} from '@/actions/developer-detail'
+import { generateHeader, URLS } from '@/constants/api'
+import { cloneableGenerator } from '@redux-saga/testing-utils'
+import { fetcher } from '@reapit/elements'
+import { developerDetailStub } from '../__stubs__/developer-detail'
+import { errorThrownServer } from '@/actions/error'
+import errorMessages from '@/constants/error-messages'
+
+jest.mock('@reapit/elements')
+
+const developerId = '72ad4ed6-0df0-4a28-903c-55899cffee85'
+const params = { data: { id: developerId } }
+
+describe('developerDetailSaga', () => {
+  it('should listen developer request data & create app action', () => {
+    const gen = developerDetailSaga()
+
+    expect(gen.next().value).toEqual(
+      takeLatest(ActionTypes.DEVELOPER_DETAIL_REQUEST_DATA, developerDetailFetchDataFetch),
+    )
+    expect(gen.next().done).toBe(true)
+  })
+})
+
+describe('developerDetail fetch data', () => {
```

Comment:
This one also has fetchMyIdentity Action, pls reuse that instead of writing the new things